src/Chat/utils/sanitizers.js

```diff
@@ -1,6 +1,35 @@
-import { replace, toLower } from 'lodash'
+import { ANONYMOUS_USERNAME } from '../constants'
 
-const oauth = (oauth = '') => `oauth:${replace(oauth, /oauth:/gi, '')}`
-const channel = (channel = '') => `#${replace(toLower(channel), /^#/g, '')}`
+function channel(value) {
+  if (value == null) {
+    return '#'
+  }
 
-export { oauth, channel }
+  if (value.startsWith('#')) {
+    return value
+  }
+
+  return `#${value}`
+}
+
+function password(value) {
+  if (value == null || value === 'SCHMOOPIIE') {
+    return 'SCHMOOPIIE'
+  }
```

Comment:
Does it have to be `SCHMOOPIIE`?

I mean Schmoopiie did amazing work for the community  😆 but would `TWITCHJS` or `TWITCH-JS` also work here?

Reply:
I guess but I need to test if that's possible. I just chose this one because Twitch does.